### PR TITLE
Login View added

### LINKED ISSUE #1: Login Page
### Note
<b>create seperate branch named-> yourname-page or component name you are working on</b>

Proper validation should be done for each field.

### Preview

image

### PR DESCRIPTION
**Pull request by Samarth-143**
https://github.com/user-attachments/assets/7130bb9b-ebbf-4550-9430-722ab6f752dd

here is a recording of the project
fix: #1

### CODE REVIEW

**Comment by SHREYANSHSK** [comment]
what help you want as you have assigned help wanted label

also add screen recording how will i know what exactly have you made and link the issue

i think you have made a view with get cli instead you have to make a page that creates 3 files view, controller, binding. refer youtube video for get cli or documentation.

so do the necessary changes

**Comment by SHREYANSHSK** [comment]
refer the previous comment that i have made and do changes according to that

File: lib/app/modules/LoginView/controllers/login_view_controller.dart
Comment (SHREYANSHSK) [inline lib/app/modules/LoginView/controllers/login_view_controller.dart:10]: add files that were created automatically by get_cli , i.e dispose method, onReady , onClose method, don't remove this methods as they are helpful to prevent memory leakges

File: lib/app/modules/LoginView/views/login_view_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/LoginView/views/login_view_view.dart]: add validation for textfields, means the login button should not work when fields are empty and should display required message (*Hint- You can use TextFormField to achieve this* ).

**Review by SHREYANSHSK — CHANGES_REQUESTED** [review]
Remove home folder, only the you are working on should be there in project structure

add the screen recording for testing the app on different devices also, check the grp i have already share the documentation about *device preview package* have a look

after this changes i will probably merge the branch

**Comment by Samarth-143** [comment]
https://github.com/user-attachments/assets/1e41e9db-d97e-4a12-ab53-358af66b252e

made the suggest changes to the design of the page and other enhancements

**Review by SHREYANSHSK — APPROVED** [review]
looks good i will merge it after resolving conflicts

**Pull request closed without merging**
